custom_components/tahoma/sensor.py

```diff
@@ -289,6 +285,21 @@
         value=lambda value: str(value).capitalize(),
         entity_registry_enabled_default=False,
     ),
+    OverkizSensorDescription(
+        key="io:PriorityLockOriginatorState",
+        name="Priority Lock Originator",
+        value=lambda value: str(value).capitalize(),
```

Comment:
What will be the default value, if there is nothing? Will str(value) be 'none'?

Reply:
Nevermind, it seems that this is catched below :-)

Reply:
Look at my screenshot :)

Reply:
Which screenshot? ;-)

Reply:
Damned, they were not updated. Should be ok now.